queries/ecma/injections.scm

```diff
@@ -12,12 +12,27 @@
  arguments: ((template_string) @graphql
    (#offset! @graphql 0 1 0 -1)))
 
+; html`<input value="hello"/>`
```

Comment:
couldn't we enable this rule for any language?

Reply:
gql tag maps to graphql language, for instance

Reply:
You don't have to remove all of them. We eventually will solve the problem somehow. For instance I have a pull request that uses filetype.lua. we could also vendor this file or use plenaries file type library. to resolve the aliases at some point. The same problem exists for markdown.

Reply:
@bennypowers any updates on that remark?

Reply:
How's that?

Reply:
@theHamsta can you please clarify, have latest changes addressed your comment?

Reply:
Do the patterns cause any performance problems? Wouldn't it be better to create predicate that resolves the languages from `"application/XXX"`?